run-tests.sh

```diff
@@ -11,4 +11,7 @@ isort -rc -c -df **/*.py && \
 check-manifest --ignore ".travis-*" && \
 sphinx-build -qnNW docs docs/_build/html && \
 python setup.py test && \
+reana-client --help > cmd_list.txt && \
```

Comment:
Works great! what do you think about moving this test before? Maybe just after `isort`, so we save some time, if CLI has changed test won't be executed, in principle it is easier to fix this documentation issue than fixing a Python test.